examples/fpn/README.md

```diff
@@ -1,31 +1,37 @@
-# Examples of Feature Pyramid Networks for Object Detection [1]
+# Examples of Faster R-CNN and related models using FPN [1, 2]
 
 ## Performance
 MS COCO 2017 Val
 
-| Model | Original | ChainerCV |
-|:-:|:-:|:-:|
-| Faster-RCNN FPN ResNet50 | 36.7 % [2] | 37.1 % |
-| Faster-RCNN FPN ResNet101 | 39.4 % [2] | 39.5 % |
+| Backbone | Supervision | Detectron (Bbox) | Ours (Bbox) | Detectron (Mask) | Ours (Mask)| 
```

Comment:
I'd like to unify the format of tables that compare our implementations with the original implementations. Currently, we are using a variety names of column, e.g. `Original/Reference/<the name of original framework>` and `ChainerCV/Ours`.

Reply:
`Original` is sometimes easier to use than the name of the project. For instance, `SegNet-Tutorial` is the name for SegNet. IMO, this is verbose.
How about unifying it to `Original` and `Ours`.

Reply:
> How about unifying it to Original and Ours.
OK. Could you update the table of FPN? The other tables should be fixed in another PR.

Reply:
OK